### DISCUSSION TITLE
I dont have a phone so i cant use the discord (needs phone verification on) but i need assistance

### DISCUSSION THREAD

**Discussion by shovelbug** [oraxen/oraxen#994] [Q&A] [answered]
I bought a thing from mcmodels for use with oraxen and i cant get the models to show up properly, the items do show up in oraxen though can someone assist me?

**Comment by Boy0000** [comment]
please DM me on discord and I will verify ur discord account so u dont need a phone number @shovelbug

**Reply by shovelbug** [reply, marked as answer]
i ended up figuring it out i wasnt merging the resource packs right plus i had to leave certain files out to successfully remove the oraxen branding from the resource pack (i might modify the files later to change the branding to match my server specifically) (as a side note i would never be confortable giving access to my discord account to anyone, if i linked my account to your phone you could steal my account anytime)
